coinx-funds.js

```diff
@@ -1,88 +1,63 @@
 'use strict';
 
-const path = require('path');
-const fs = require('fs');
-const program = require('commander');
+const _ = require('lodash');
+const coinx = require('./coinx-core');
 const chalk = require('chalk');
-const homedir = require('homedir');
+const program = require('commander');
 const capitalize = require('capitalize');
 const inquirer = require('inquirer');
 const columnify = require('columnify');
-const _ = require('lodash');
-
-const Poloniex = require('./lib/poloniex');
-const Liqui = require('./lib/liqui');
-const Bittrex = require('./lib/bittrex');
-const Bitfinex = require('./lib/bitfinex');
-const Kraken = require('./lib/kraken');
-
 const cryptocompare = require('./lib/cryptocompare');
+const coinsLookup = coinx.coins();
+const config = coinx.config();
+const exchanges = coinx.exchanges();
+let exchangesToCheck = [];
 
-const coinxHome = path.join(homedir(), 'coinx');
-const keyFilePath = path.join(coinxHome, 'coinx.json');
-const coinListPath = path.join(coinxHome, 'coinList.json');
-
-let coinLookup = {};
-try {
-	coinLookup = require(coinListPath);
-} catch (e) {
+if (Object.keys(coinsLookup).length === 0) {
 	console.error(chalk.red('Please run `coinx update` to get the latest list of coins.'));
 	process.exit(1);
 }
 
-if (!fs.existsSync(keyFilePath)) showNotConfigured();
-
-let keys = require(keyFilePath);
-if (!Object.keys(keys).length) showNotConfigured();
+if (Object.keys(config).length === 0) {
+	console.log(chalk.red('Need to configure at least one exchange.'));
+	console.log(chalk.red('Run \'coinx configure [name of exchange]\''));
+	process.exit(1);
+}
 
 program
 	.option('-e, --exchange [name]', 'Get balances at the specified exchange.')
 	.option('-a, --alphabetically', 'Sort the balance list alphabetically.')
 	.option('-n, --numerically', 'Sort the balance list by the number of coins, descending.')
-	.option('-v, --value', 'Sort the balance list by the value of each coin in US dollars, descending.')
```

Comment:
Just noticed that this line is removed, most likely caused by a bit painful merge conflict I had, feel free to revert it.